kolibri/plugins/oidc_provider_plugin/settings.py

```diff
@@ -0,0 +1,2 @@
+INSTALLED_APPS = ["oidc_provider"]
+OIDC_LOGIN_URL = "/user/#/signin"
```

Comment:
Note, switched from `LOGIN_URL` to `OIDC_LOGIN_URL`

Will this work as intended though? I'm unsure of how things will be affected by our language prefixes, and also potentially by the client-side routing

Reply:
nope, it has to be `LOGIN_URL` . The provider library requires this name. We can do some monkey patching if you want to add a prefix.